test/test_flake8.py

```diff
@@ -26,7 +26,7 @@
 
 def test_flake8():
     style_guide = get_style_guide(
-        extend_ignore=['D100', 'D104'],
+        extend_ignore=['D100', 'D104', 'W605'],
```

Comment:
@clalancette why ignore `Ẃ605`?

Reply:
Ah, this one is tricky and I should have put a comment for it.

I think flake8 is misunderstanding the triple-quoted string at https://github.com/ros-infrastructure/rosdoc2/blob/b8f62374d74580662f617ff4ea01df5222252e81/rosdoc2/verbs/build/builders/sphinx_builder.py#L27-L131 , and thinking that there is a `\.` in a regex without an `r` at line https://github.com/ros-infrastructure/rosdoc2/blob/b8f62374d74580662f617ff4ea01df5222252e81/rosdoc2/verbs/build/builders/sphinx_builder.py#L104 .  So it was always giving me the [W605](https://www.flake8rules.com/rules/W605.html) warning.

That said, disabling it completely is indeed too big of a hammer.  I'll change it to just ignore W605 on that one line.

Reply:
@clalancette fyi, one alternative solution here would be to use raw triple-quoted strings `r"""..."""`.

Reply:
Yeah, that's true.  I was a little hesitant to make changes that way, since there are 2 levels of replacement being done; one when we write out the string to a file from within `rosdoc2` (using f-strings), and then another at runtime when sphinx actually executes this file.  I *think* what you've suggested will work, but nothing is safer than a comment :).